iban-tools.gemspec

```diff
@@ -19,8 +19,6 @@ Gem::Specification.new do |s|
   ]
   s.description  = "Validates IBAN account numbers"
 
-  # s.add_dependency "banking_data" # from Github, see Gemfile
```

Comment:
Any reason why you removed this? I'm for readding. It helps a bit the readability

Reply:
I found it a bit confusing because now the repo isn't in the Gemfile anymore but in the ones generated by appraisal, but I added a comment to explain